demo/enhanced-code-blocks.md

```diff
@@ -198,7 +198,7 @@ Console blocks have special handling. Try clicking the line number next to line
 
 <!-- prettier-ignore-start -->
 ```console
-$ pwd
+$ pwd # prints the current working directory
```

Comment:
I decided to let these comments be included in the copied commands. I think it could be useful for students as they scroll through their command history.